packages/components/webpack.config.js

```diff
@@ -106,5 +106,10 @@ module.exports = {
         }),
         new IgnorePlugin(/^\.\/locale$/, /moment$/)
     ],
-    externals: ['react', 'react-dom', 'reactn', 'react-bootstrap', 'immutable', 'jquery']
+    externals: [
+        'react', 'react-dom', 'reactn', 'react-bootstrap', 'immutable', 'jquery',
+        '@labkey/api', 'react-treebeard', 'react-beautiful-dnd', 'react-datepicker', 'react-router',
+        'react-bootstrap-toggle', 'date-fns', 'numeral', 'font-awesome', 'formsy-react', 'formsy-react-components',
+        'moment', 'moment-jdateformatparser', 'moment-timezone', 'lodash', 'history',
```

Comment:
We should add at least two more packages: `react-redux`, and `redux-actions`. We maybe want to add `xhr-mock` as well, but only if we export any code that uses it in this repo (I am not sure we do, but maybe you know of something).

Reply:
done